### PR TITLE
Docker 2

### PR DESCRIPTION
**Pull request by mattkrick**
GOALS:
- [ ] can `git pull`, run a single command, and begin developing
- [ ] can run a single command that creates a production-ready container

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -29,6 +30,8 @@ services:
     volumes:
       - .:/usr/src/app
       - app_node_modules:/usr/src/app/node_modules
+      - app_build:/usr/src/app/build
```
Comment (jordanh) [inline docker-compose.yml]: One option for a better way: any assets that are built by the repo but are ultimately system independent, could be built before the container build cycle and then copied into the container.

One thing we'd have to be sensitive about is how to handle any CDN paths in the built assets.

**Comment by humenius** [comment]
Are there any news on this one?

**Comment by mattkrick** [comment]
hey @humenius!

this is stuck behind hiring a new backend engineer & moving to k8s : https://www.parabol.co/join/senior-backend-developer

**Comment by nbartels** [comment]
I rebased the docker 2 branch on the current master and made some minor modifications to get everything built, but it is still not running. I get an exception in the babel module. Sadly I'm not a node/JS developer, so this is a bit harder to resolve. 
Maybe you like to check the branch in my repo: https://github.com/nbartels/parabol/tree/docker-2

**Comment by mygalus** [comment]
@nbartels, I have requested a new branch from the master for doing this, I put a coworker on it and he decide to restart from an alpine linux as a base image for it. I have review it and add few little helper tools in it, if you have time, could you review it ? [Update Dockerfile / docker-compose](https://github.com/ParabolInc/parabol/pull/3765) , any comment are welcome.
Thanks a lot !

**Comment by mattkrick** [comment]
gone stale, moving devops outside this repo

**Pull request closed without merging**